### PR TITLE
ChatOps on RHEL7 + Enterprise on Ubuntu/RHEL7

### PR DESCRIPTION
**Pull request by dzimine**
Tested and added/adjusted docs on Chatops and Enterprise, on RHEL7 and Ubuntu.

### CODE REVIEW

**Comment by lakshmi-kannan** [comment]
Will you be making changes to the script as well? 
https://github.com/StackStorm/st2-packages/pull/200

**Comment by lakshmi-kannan** [comment]
Are you going to hold the PR until @DoriftoShoes has a way to generate licenses? This should've been two PRs. One for chatops and one for enterprise. Oh well.

**Comment by dzimine** [comment]
> Will you be making changes to the script as well? 

please take this.

> Are you going to hold the PR until @DoriftoShoes has a way to generate licenses? This should've been two PRs. One for chatops and one for enterprise. Since this is to master, we might be fine with a merge?

Right. Because it's a master I won't hold. But yes, good catch. At least we won't move it to v1.3 even if we move chatops (they areseparate commits).

File: docs/source/install/rhel7.rst [hunk 1]
```diff
@@ -284,31 +284,35 @@ Use your browser to connect to ``https://${ST2_HOSTNAME}`` and login to the WebU
...
+    # Create notification rule if not yet enabled
+    st2 rule get chatops.notify || st2 rule create /opt/stackstorm/packs/chatops/rules/notify_hubot.yaml)
 
-* Install docker:
+* `Install NodeJS v4 <https://nodejs.org/en/download/package-manager/>`_): ::
 
-    `Docker install for RHEL 7 <https://docs.docker.com/engine/installation/linux/rhel/>`_.
-    `Docker install for CentOS 7 <https://docs.docker.com/engine/installation/linux/centos/>`_.
+      curl -sL https://rpm.nodesource.com/setup_4.x | sudo -E bash -
+      sudo yum install -y gcc-c++ make
```
Comment (lakshmi-kannan) [inline docs/source/install/rhel7.rst]:
```
## Run `yum install -y nodejs` (as root) to install Node.js 4.x LTS Argon and npm.
## You may also need development tools to build native addons:
##   `yum install -y gcc-c++ make`
```

We don't need this.

**Pull request merged**